cirq/protocols/mixture.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Protocol for objects that are mixtures (probabilistic combinations)."""
+
+from typing import Any, Iterable, Tuple, Union
+
+from typing_extensions import Protocol
+
+import numpy as np
+
+from cirq.type_workarounds import NotImplementedType
+
+# This is a special indicator value used by the inverse method to determine
+# whether or not the caller provided a 'default' argument.
+RaiseTypeErrorIfNotProvided = (None,)
```

Comment:
Use `([],)` instead. In principle an implementation of python could make all expressions `(None,)` point to the same reference, so the caller could make this.

Reply:
Just copying your pattern ;)  Changed.

Reply:
Ah saw we moved away from the (None,).